src/App.js

```diff
@@ -130,26 +131,30 @@ class Bloc extends Component {
 
 const ColorButts = ({ background, onChangeBackground }) => {
   return (
-    <ul className="ColorButts">
-      change background :
-      {backgroundColors.map((bg, index) => {
-        const colorButtsStyle = {
-          background: bg.color,
-          color: Color(bg.color).isDark() ? "#FFF" : "#000"
-        };
-        console.log(background, bg.color)
-        return (
-          <li
-            key={index}
-            onClick={()=>onChangeBackground(bg.color)}
-            className={background === bg.color ? "selected" : ""}
-            style={colorButtsStyle}
-          >
-            {bg.color}
-          </li>
-        );
-      })}
-    </ul>
+    <div className="ColorButts">
+      <ul>
+        change background :
+        {backgroundColors.map((bg, index) => {
+          const colorButtsStyle = {
+            background: bg.color,
+            color: Color(bg.color).isDark() ? "#FFF" : "#000"
+          };
+          return (
+            <li
+              key={index}
+              onClick={()=>onChangeBackground(bg.color)}
+              className={background === bg.color ? "selected" : ""}
+              style={colorButtsStyle}
+            >
+              {bg.color}
+            </li>
+          );
+        })}
+      </ul>
+      <div className="ColorPickerContainer">
+        <HuePicker color={background} onChange={(color)=>onChangeBackground(color.hex)} />
```

Comment:
```suggestion
        <HuePicker color={background} onChange={(color) => onChangeBackground(color.hex)} />
```
Missing space

Reply:
thanx @jonny, but i will have to close this for now 
https://github.com/nakulrathore/Box-Shadows/pull/21#issuecomment-537919962